parlai/agents/drqa/drqa.py

```diff
@@ -83,13 +99,21 @@ def add_to_dict(self, tokens):
 # ------------------------------------------------------------------------------
 
 
-class DocReaderAgent(Agent):
+class DrqaAgent(Agent):
 
     @staticmethod
     def add_cmdline_args(argparser):
         config.add_cmdline_args(argparser)
+        SimpleDictionaryAgent.add_cmdline_args(argparser)
 
-    def __init__(self, opt, shared=None, word_dict=None):
+    @staticmethod
+    def dictionary_class():
```

Comment:
I guess it does make it so that you can override this param directly from the command line instead of changing this function, which maybe some people would want to do